lib/lndclient/LndClient.ts

```diff
@@ -72,6 +73,25 @@ class LndClient extends BaseClient {
     });
   }
 
+  private connect = (lndCLient: LndClient, subscribeInvoices: boolean = false) => {
+    const { host, port } = lndCLient.config;
+
+    lndCLient.lightning = new LightningClient(`${host}:${port}`, lndCLient.credentials);
+
+    lndCLient.getInfo().then((getInfoResponse) => {
+      if (getInfoResponse && getInfoResponse.numActiveChannels >= 1) {
+        lndCLient.setStatus(ClientStatus.CONNECTION_VERIFIED);
+        if (subscribeInvoices) {
```

Comment:
In my experience, it's necessary to resubscribe to invoices whenever a connection to lnd is reestablished, so we might want to remove this if statement and just call `subscribeInvoices` every time we connect.

Reply:
That was my first approach, fixing it again.

Reply:
@sangaman Are you sure? This methods gets also called from the constructor which means always being subscribed to the invoices. `subscribeInvoices` should be private then.